spec/modules/helpers.rb

```diff
@@ -1,5 +1,12 @@
 require 'rspec/expectations'
 require_relative "../matchers/category_matchers"
+require_relative "../matchers/discussion_mathcers"
+require_relative "../matchers/shared_matchers"
+
+CATEGORY_NAME = "Test Category Name"
+CATEGORY_DESCRIPTION = "Test category description"
+DISCUSSION_TITLE = "Test Discussion Title"
+DISCUSSION_BODY = "Test discussion body"
```

Comment:
The use of constants here seems overly rigid.  The structure of this module with constants means you cannot reuse the module helpers or test with different categories.  This limits the usefulness of separating the logic into reusable matchers.  Perhaps parameterizing the values and using these constants as defaults could make this less brittle?